ui-components/src/overlays/Modal.tsx

```diff
@@ -31,13 +31,14 @@ export const Modal = (props: ModalProps) => {
   return (
     <Overlay
       ariaLabel={props.ariaLabel || props.title}
+      aria-labelledby={props.ariaLabel || props.title}
```

Comment:
I _think_ we want this label on the modal div below and not on the overlay, but I could be wrong here (@slowbot )